pkg/server/ptypes/project_template.go

```diff
@@ -0,0 +1,117 @@
+package ptypes
+
+import (
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+	"github.com/imdario/mergo"
+	"github.com/mitchellh/go-testing-interface"
+	"github.com/stretchr/testify/require"
+
+	"github.com/hashicorp/waypoint/internal/pkg/validationext"
+	pb "github.com/hashicorp/waypoint/pkg/server/gen"
+)
+
+const (
+	PROJECT_TEMPLATE_ID_LENGTH               = 64
+	PROJECT_TEMPLATE_NAME_LENGTH             = 64
+	PROJECT_TEMPLATE_TAG_LENGTH              = 64
+	PROJECT_TEMPLATE_SUMMARY_LENGTH          = 64
```

Comment:
The initial design number was 110. Maybe this is too short?

Reply:
Oh this is a good call - if anything, it'll be longer post base64 encoding!